src/runway/tests/handlers/yaml_lint.py

```diff
@@ -28,13 +30,29 @@ def get_yaml_files_at_path(provided_path):
         )
         return yaml_files + yml_files
 
+    def get_yamllint_options(self, path, quote_paths=True):
```

Comment:
should this be a `@classmethod` or even a `@staticmethod`? since it doesn't care about the state of the instance it wouldn't need `self`. it also doesn't care about the state of the class so it wouldn't need `cls` other than to call other static methods.